### PR TITLE
Samyak testing PR

### PR DESCRIPTION
**Pull request by noviodelangel**

### CODE REVIEW

File: InnoTopicWebsite/src/app/skills/work-experience.component.html [hunk 1]
```diff
@@ -128,9 +116,9 @@ <h2 class="section-title">Experience <!--<span class="comment">(hover/tap to hig
   <!--    <app-skill-levels-icons [skillLevels]="{have: 'beginner', want: 'expert'}"></app-skill-levels-icons>-->
   <!--    <app-skill-levels-icons [skillLevels]="{have: 'advanced'}"></app-skill-levels-icons>-->
       <p class="example">Example:</p>
-      <app-topic-skill [tId]="'Python'"
+      <app-topic-skill [tId]="'Angular'"
```
Comment (noviodelangel) [inline InnoTopicWebsite/src/app/skills/work-experience.component.html]: @samyakjainnnnn @karol-depka  says it should stay python - do you know if it was a manual update or a merge conflict resolution issue?

File: InnoTopicWebsite/src/app/work-projects/work-projects-data.ts [hunk 2]
```diff
@@ -61,10 +61,10 @@ export class WorkOrganisationsAndProjectsData {
     logoFileName: `epic-elite-logo.svg`,
     description: 'Test #Rust #React and #AWS hashtags',
     projects: {
-      'AI-based LLM-powered processing of product metadata': project({
+      'EpicElite Platform': project({
```
Comment (noviodelangel) [inline InnoTopicWebsite/src/app/work-projects/work-projects-data.ts]: Hmmm @samyakjainnnnn EpicElite is also present below in line 131 - why it was modified here?

File: InnoTopicWebsite/src/app/work-projects/work-projects-data.ts [hunk 3]
```diff
@@ -131,7 +131,7 @@ export class WorkOrganisationsAndProjectsData {
       'EpicElite Platform': project({
         roles: 'Lead/Senior Full-Stack Developer & Interviewing and Recruiting (Consultant)',
         description: 'Recruited and interviewed 5 Developers and a UI/UX/Web/Graphics Designer. I have lead a team of 5 Developers. Did code&architecture reviews.' +
-          'Working directly with CEO & Director of Operations. Communication in English, Spanish, Polish. #Python #Django + #TypeScript app using #MongoDB and #MariaDB .',
+          'Working directly with CEO & Director of Operations. Communication in English, Spanish, Polish.',
```
Comment (noviodelangel) [inline InnoTopicWebsite/src/app/work-projects/work-projects-data.ts]: @samyakjainnnnn above hashtags were placed there on purpose so I guess they should stay there

File: InnoTopicWebsite/src/assets/images/logos/recaptcha-icon.svg [hunk 4]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (noviodelangel) [inline InnoTopicWebsite/src/assets/images/logos/recaptcha-icon.svg]: @samyakjainnnnn what was the reason for removing svg files?

**Comment by JayTailor45** [comment]
@karol-depka Closing this as we no longer need this change. Thanks!

**Pull request closed without merging**